src/prommis/HD_CE.py

```diff
@@ -0,0 +1,344 @@
+from pyomo.environ import ConcreteModel, Constraint, Param, Var, units,PositiveReals, log
```

Comment:
This is not the right place for this file. If it is costing related then it should go in a costing folder somewhere.